Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,95 +1,143 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
-        /// <summary>
-        /// Sort the array in ascending order.
-        /// If array empty or null- don't do anything.
-        /// </summary>
-        /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (IsNullArray(array))
+                return;
+            
+            int temp;
+
+            for (int i = 0; i < array.Length; i++)                  //For each index of the array...
+            {
+                for (int j = i + 1; j < array.Length; j++)  
+                {
+                    if (array[j] < array[i])                        //Check if each following index is smaller.
+                    {
+                        temp = array[i];
```

Comment:
Swapping is repeated 2 times at least, you can make a function Swap, can't you :) ?

Reply:
Swap(array, index1, index2)